.gitignore

```diff
@@ -0,0 +1,93 @@
+# Created by .ignore support plugin (hsz.mobi)
```

Comment:
Yea, if you'd like it removed -- can do. We've used a rather broad coverage on .gitignore in our group in the past because different system configurations and not everyone notifies when some tool dumps an extra file.

I think some .gitignore file should be included however for now I will delete it. For example, when running unit tests a file for coverage results `.coverage` is generated. For those contributing to this project, they may not notice that file be added and accidentally commit it.